Symfony/CS/Tokenizer/Token.php

```diff
@@ -99,9 +100,22 @@ public function equals($other)
         $selfPrototype = $this->getPrototype();
 
         foreach ($otherPrototype as $key => $val) {
-            if ($selfPrototype[$key] !== $val) {
+            // Make sure the token has such key
```

Comment:
It's not a `rule for comment`, it's more a matter of grammar.
Either something is a sentence (then starting uppercased and ending with punctuation mark), either not.
Last, but not least, keep consistency with rest of code.